example/spec/helpers/granuleUtils.js

```diff
@@ -145,29 +144,20 @@ async function setupTestGranuleForIngest(bucket, inputPayloadJson, granuleRegex,
 }
 
 const deleteGranules = async (prefix, granules) => {
-  process.env.GranulesTable = `${prefix}-GranulesTable`;
-  const granuleModel = new Granule();
-  return await Promise.all(
+  await Promise.all(
     granules.map(async (granule) => {
-      // Temporary fix to handle granules that are in a bad state
-      // and cannot be deleted via the API
-      try {
-        if (granule.published === true) {
-          return await removePublishedGranule({
-            prefix,
-            granuleId: granule.granuleId,
-          });
-        }
-        return await deleteGranule({
+    // Temporary fix to handle granules that are in a bad state
```

Comment:
Some questions here:
1. what was/is the 'Temporary fix', and is it still needed?
2. was the `try...catch` related to the Temporary fix? curious what the reason is that it is no longer needed
3. not important, but I am so curious what the `TODO wat` was all about

Reply:
`what was/is the 'Temporary fix', and is it still needed?`
I could be wrong but my understanding is that this temporary fix addresses the need for granules to be unpublished from CMR before being deleted, and that logic is still in place? Though I say I could be wrong because that functionality seems permanent rather than temporary 🤔 

`was the try...catch related to the Temporary fix? curious what the reason is that it is no longer needed`
Yes? The reason the try/catch is no longer needed is because we no longer need to delete the granule manually from DynamoDB